aten/src/ATen/native/native_functions.yaml

```diff
@@ -2418,6 +2418,19 @@
 
 - func: sigmoid.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: logit(Tensor self, float? eps=0.0) -> Tensor
```

Comment:
why make it optional and give it a non-optional default value?  AFAICT, this just lets you write `torch.logit(self, eps=None)`, but why would I want to do that?

Reply:
Done.